src/emails/templates/breachAlert/BreachAlertEmail.tsx

```diff
@@ -73,17 +75,6 @@ export type RedesignedBreachAlertEmailProps = {
   breachedEmail: string;
   utmCampaignId: string;
   subscriber: SubscriberRow;
-  /**
-   * We need to run a bunch of queries to collect this data,
-   * so it's optional; however, make sure to pass it in for
-   * free users who are eligible for Plus (i.e. in the US),
-   * who have run a scan — those are the ones we show a
-   * <DataPointCount> for at the moment. We also use the
-   * presence of this property to determine whether free
-   * US users have run a scan. If, in the future, we assume
-   * we can do the same for Plus users, that might break, but
-   * that's a risk we'll have to take for now:
-   */
```

Comment:
I think this comment is relevant again, except maybe the part about determining whether a scan has run, which you replaced with the (better) checking for a OneRep profile ID:

```suggestion
  /**
   * We need to run a bunch of queries to collect this data,
   * so it's optional; however, make sure to pass it in for
   * free users who are eligible for Plus (i.e. in the US),
   * who have run a scan — those are the ones we show a
   * <DataPointCount> for at the moment.
   */
```